client/landing/stepper/declarative-flow/internals/steps-repository/site-migration-application-password-authorization/index.tsx

```diff
@@ -14,15 +15,15 @@ import { useDispatch } from 'calypso/state';
 import { resetSite } from 'calypso/state/sites/actions';
 import Authorization from './components/authorization';
 import useStoreApplicationPassword from './hooks/use-store-application-password';
-import type { Step } from '../../types';
+import type { Step as StepType } from '../../types';
```

Comment:
The alias seems redundant,  the keyword type is already describing is a type

Reply:
that's there because we are importing a `Step` here
https://github.com/Automattic/wp-calypso/pull/102305/files#diff-2dce034d00bfd12e67b9628930b119dc8d708633900fae1cc3c50e316a5a86e5R1